testing/features/support/pages/breadcrumbs.rb

```diff
@@ -2,9 +2,9 @@
 
 module DesignSystem
   class Breadcrumbs < SitePrism::Page
-    set_url "/iframe.html?id=3-components--breadcrumb&viewMode=story"
+    set_url "/iframe.html?id=components-breadcrumbs--breadcrumbs-default&viewMode=story"
 
-    section :initial_form, "#a11yComponentToTest" do
+    section :initial_form, ".cads-breadcrumbs" do
```

Comment:
This "might" be the issue, but I need a bit more context.

Also I'm guessing this is a breaking change. Unfortunately due to our infrastructure we can't test things until after they're released. So it's likely you'll need to slap a `@future_release` tag on the feature with `@v1.11.0+`